app/src/main/java/com/ecosystem/kin/app/AccountInfoServiceImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.ecosystem.kin.app;
+
+import com.ecosystem.kin.app.model.SignInRepo;
+import com.kin.ecosystem.Kin;
+import com.kin.ecosystem.common.KinCallback;
+import com.kin.ecosystem.common.exception.KinEcosystemException;
+import com.kin.ecosystem.common.model.WhitelistData;
+import com.kin.ecosystem.transfer.AccountInfoService;
+/*
+*  Example usage of extending AccountInfoService
+*  Name of class should not be changed
```

Comment:
Why ? is it explained somewhere ?

Reply:
because we wake service form the other side 
the name of the service is the same for all app
its the package that is changed
we can update clearer explanation